cmd/tailscale/cli/debug.go

```diff
@@ -734,3 +739,18 @@ func runSetExpire(ctx context.Context, args []string) error {
 	}
 	return localClient.DebugSetExpireIn(ctx, setExpireArgs.in)
 }
+
+func runCapture(ctx context.Context, args []string) error {
+	stream, err := localClient.StreamDebugCapture(ctx)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintln(os.Stderr, "To examine in wireshark, run:")
+	fmt.Fprintf(os.Stderr, "\t%s | wireshark -X lua_script:<path to ts-dissector.lua> -k -i -\n", strings.Join(os.Args, " "))
+	fmt.Fprintln(os.Stderr, "Dont forget to download the Tailscale debug dissector! You can get it from the Tailscale source tree in wgengine/capture.")
```

Comment:
why not go:embed it and let this tool write it out as needed? It could even start wireshark.

Reply:
Ooooo I like this. Can a sandboxed MacOS binary fork out like that, and can the exec'd program access tempfiles of its parent?